### PR TITLE
CANS-953: AKA shown in search results if matching to search term

### PR DESCRIPTION
**Pull request by LeonidMarushevskyi**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-953

## Description

AKA shown in search results if matching to search term. Implemented in a similar manner to Intake https://osi-cwds.atlassian.net/browse/SNAP-817. Fuse.js library used to match AKA on frontend.

**Before** 
before
**After**
after

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    </div>
+  ) : (
+    ''
+  )
+}
+AkaInfo.defaultProps = {
+  aka: {},
+}
+AkaInfo.propTypes = {
+  aka: PropTypes.object,
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js]: If the component is directly using properties on an object, we should define that shape in the propTypes. It helps future readers understand at a glance what data is required.

**Comment by LeonidMarushevskyi** [comment]
> Look good to me. Just curious, do we need any changes or additions in regression test ?

Spoke with Stan regarding this question and we decided not to develop any regression tests for this functionality because we don't have test data for this use case.

File: app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { isEmpty } from '../../../../../util/common'
+
+const AkaInfo = ({ aka }) => {
+  return aka && !isEmpty(aka) ? (
```
Comment (denys-davydov) [inline app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js]: Small note: if you have `isEmpty(aka)`, you don't need the `aka && ` part.

File: app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js (hunk 1 above)
Comment (LeonidMarushevskyi) [reply app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js]: Fixed

File: app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js (hunk 2 above)
Comment (LeonidMarushevskyi) [reply app/javascript/Application/components/Search/ClientSearch/QueryResults/PersonSuggestion/AkaInfo.js]: Fixed

**Pull request merged**